src/main/g8/$name__norm$-impl/src/main/resources/application.conf

```diff
@@ -32,6 +32,7 @@ akka.cluster.sharding.state-store-mode = ddata
 # messages to avoid the use of Java serialization.
 akka.actor.serialization-bindings {
   "akka.Done"                 = akka-misc
+  "akka.NotUsed"              = akka-misc
```

Comment:
NotUsed was missing but was added together with the others in Akka 2.5.8

Reply:
Why is this required now? It hasn't failed so far, right?

Reply:
It is not required, but like the other is the recommended way of doing that. For the same reason we have the others.

Without it, those types will be serialized using java serialization.